src/Rings/mpoly-localizations.jl

```diff
@@ -356,7 +356,7 @@ function prime_ideal(S::MPolyComplementOfKPointIdeal)
   return S.m
 end
 
-function is_prime(P::Hecke.ZZIdl) 
+function is_prime(P::Hecke.ZZIdl)  # TODO: move to Hecke
```

Comment:
```suggestion
function is_prime(P::Hecke.ZZIdl)  # TODO: move to Hecke
```
Did you move this already?

Reply:
That's in https://github.com/thofma/Hecke.jl/pull/1274 which has not yet been merged, let alone removed; the marker here is meant to make it easy to grep for "TODO.*move" to find places that can now be removed...

Reply:
"Type piracy" aka using julia as intended originally (extending methods).